weed/storage/needle_map_file_pool_test.go

```
@@ -0,0 +1,188 @@
+			target, err := os.Readlink(filepath.Join("/proc/self/fd", e.Name()))
+			if err != nil {
+				continue // raced with a close
+			}
+			if isIndexFileUnder(target, prefix) {
+				count++
+			}
+		}
+		return count, true
+	}
+
+	out, err := exec.Command("lsof", "-p", strconv.Itoa(os.Getpid()), "-F", "n").Output()
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)